src/main/java/org/example/Data.java

```diff
@@ -0,0 +1,14 @@
+package org.example;
+
+import java.math.BigDecimal;
+
+public class Data {
```

Comment:
Dane czego?

Reply:
Static constants used in the program.
resolved.